tests/telemetry/test_telemetry.py

```diff
@@ -45,23 +47,43 @@ def setup_telemetry_forpyclient(duthost):
     else:
         logger.info('client auth is false. No need to restart telemetry')
 
-def verify_telemetry_dockerimage(duthost):
+@pytest.fixture(scope="module", autouse=True)
+def verify_telemetry_dockerimage(duthosts, rand_one_dut_hostname):
```

Comment:
@wangxin This fixture function will be invoked before running each test case. Please have a review!